ouroboros-network/cardano-diffusion/Ouroboros/Cardano/Network/Diffusion/Types.hs

```diff
@@ -0,0 +1,63 @@
+{-# LANGUAGE DataKinds      #-}
+{-# LANGUAGE GADTs          #-}
+{-# LANGUAGE KindSignatures #-}
+
+module Ouroboros.Cardano.Network.Diffusion.Types where
```

Comment:
Could you also export this module from `Ouroboros.Cardano.Network.Diffusion` if it's not done already, e.g.
```
module ...
 ( ...
 , module Diffusion.Types
 , ...
 ) where

```